README.md

```diff
@@ -5,15 +5,6 @@ Currently only useful for [VIKTOR](https://www.viktor.ai) applications.
 # Installation
 Paste the link of this repository in your `requirements.txt`:

```

Comment:
Pasting the link is no longer relevant, we should just add the dependency:

```suggestion
Add the `excel-graph-parser` dependency  in your `requirements.txt`:
```
Comment